src/lib/support/tests/TestJsonToTlv.cpp

```diff
@@ -317,6 +317,26 @@ int Initialize(void * apSuite)
     return SUCCESS;
 }
 
+void TestJsonToTlvWithoutStruct(nlTestSuite * inSuite, void * inContext)
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+    TLV::TLVReader reader;
+    bool expectedVal       = false;
+    std::string jsonString = "{\n"
+                             "   \"1:BOOL\" : true\n"
```

Comment:
There should be tests for ids outside the uint8 range, for both directions.

Reply:
updated, thanks